modules/mod_logging/mod_logging.erl

```diff
@@ -201,6 +206,33 @@ code_change(_OldVsn, State, _Extra) ->
 %% support functions
 %%====================================================================
 
+%% @private Check the health of the db pool. When usage is to high a warning will be %% put in the log.
+check_db_pool_health(Context) ->
+    Site = z_context:site(Context),
+    Advice = "please increase the pool size.",
+
+    case exometer:get_value([zotonic, Site, db, pool_full], one) of
+        {ok, [{one, FullCounts}]} when FullCounts > 0 ->
+            z:error("Database pool is exhausted, ~s",
+                    [Advice],
+                    [{module, ?MODULE}, {line, ?LINE}],
+                    Context);
+        {ok, _} ->
+            ok;
+        {error, not_found} ->
+            ok
+    end,
+
+    case exometer:get_value([zotonic, Site, db, pool_high_usage], one) of
+        {ok, [{one, HighCounts}]} when HighCounts > 0 ->
+            z:info("Database pool usage is close to exhaustion, ~s", [Advice],
```

Comment:
What if the pool is full - can `z:info` handle that?
Or do we need to add a check in the db part of the logger to not log to the db if the pool is exhausted?

Reply:
Hmm... the pool usage can fluctuate, depending on what happens. I assume I can log here, as the event is, on average 30 second old.